docs/ui/customization/fragments/react-native/theming.md

```diff
@@ -21,7 +21,7 @@ const MyTheme = {
 <Authenticator theme={MyTheme} />
 ```
 
-For React Native, you must override properties defined in AmplifyTheme.js [here](https://github.com/aws-amplify/amplify-js/blob/main/packages/aws-amplify-react-native/src/AmplifyTheme.js)
+For React Native, you must override properties defined in AmplifyTheme.js [here](https://github.com/aws-amplify/amplify-js/blob/main/packages/aws-amplify-react-native/src/AmplifyTheme.ts)
```

Comment:
Thanks for sending this in!

I've made a suggestion that links `AmplifyTheme.ts` instead of `here`. What do you think?

```suggestion
For React Native, you must override properties defined in [`AmplifyTheme.ts`](https://github.com/aws-amplify/amplify-js/blob/main/packages/aws-amplify-react-native/src/AmplifyTheme.ts)
```